lib/rubygems/specification.rb

```diff
@@ -2591,6 +2591,7 @@ def traverse trail = [], visited = {}, &block
 
   def validate packaging = true
     @warnings = 0
+    require 'rubygems/util/spdx' unless defined?(Gem::SPDX)
```

Comment:
@tenderlove, I moved the `require` here, and added a condition to hush redefinition warnings during testing.

I couldn't find this idiom elsewhere in the codebase. Please let me know if I've gone about this the wrong way.